src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -66,6 +67,13 @@ public void init() throws ServletException {
   // TODO: Check and see if lectureURL is already in database and if it is valid.
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Can you include @Override?

Reply:
@override is included right above the todo

Reply:
Oops sorry I must have missed it.